core/test/mpi/distributed/CMakeLists.txt

```diff
@@ -1 +1 @@
-ginkgo_create_test(matrix MPI_SIZE 3)
+ginkgo_create_test(matrix MPI_SIZE 1)
```

Comment:
Is this change normal? It doesn't seem related to the PR here.

Reply:
That test actually doesn't use any communication, so it can run on only one process.